### DISCUSSION TITLE
Caching

### DISCUSSION THREAD

**Discussion by rizafahmi** [ngobrolin/web#17] [Ideas]
Membahas teknik yang seringkali berguna untuk scaling

**Comment by rizafahmi** [comment]
@ivankristianto @ekafyi buat episode selasa besok bahas caching yuk

**Reply by ekafyi** [reply]
Boleh, setuju bahas caching.

**Reply by ivankristianto** [reply]
Web cache kan maksudnya? Bukan backend cache. Kl iya. Yuks.

**Reply by rizafahmi** [reply]
@ivankristianto tadinya sih maksudnya cache dari semua sisi. Tapi kalo mau bahas web cache dulu boleh juga

**Reply by rizafahmi** [reply]
Lagi rame juga nih, Redis baru mengumumkan [lisensi ganda](https://redis.com/blog/redis-adopts-dual-source-available-licensing/)

**Reply by ivankristianto** [reply]
O iya ya, boleh juga. semua cache mulai dari backend sampai frontend

**Comment by rizafahmi** [comment]
https://youtube.com/live/m7rsN2Wmb78
